sources/packages/backend/libs/test-utils/src/factories/offering-change-request.ts

```diff
@@ -0,0 +1,31 @@
+import { EducationProgramOffering, OfferingStatus } from "@sims/sims-db";
+
+/**
+ * Create a fake offering request change.
+ * @param relations dependencies:
+ *  - `currentOffering` current offering, which is requested for the changed.
```

Comment:
Maybe it would be good to mention that it is expected to be already saved to the DB, right?